packages/playwright-core/src/server/injected/roleUtils.ts

```diff
@@ -461,6 +461,60 @@ export function getElementAccessibleDescription(element: Element, includeHidden:
   return accessibleDescription;
 }
 
+// https://html.spec.whatwg.org/multipage/input.html#the-input-element
+export const kInputValidityAttributes = ['required', 'pattern', 'min', 'max', 'minlength', 'maxlength', 'step', 'type', 'value'];
```

Comment:
nit: there is no need to export all the constants/functions from this file. Just export `getElementAccessibleErrorMessage()`.